src/org/zaproxy/zap/ZAP.java

```diff
@@ -87,7 +88,9 @@ public static void main(String[] args) throws Exception {
         } catch (final Exception e) {
         	// Cant use the CommandLine help here as the 
         	// i18n messages wont have been loaded
-            System.out.println("Failed due to invalid parameters. Use '-h' for more details.");
+            System.out.println("Failed due to invalid parameters: " + Arrays.toString(args));
+            e.printStackTrace();

```

Comment:
Probably not given that Exception is being thrown for parsing errors. I'll change that to just print the exception message.